integration-tests/initiateDataRequest.spec.ts

```diff
@@ -192,6 +198,61 @@ describe('Submit a PII request with approved ticket data', () => {
     })
   })
 
+  describe('valid requests for standard and glacier copy - analysis bucket empty', () => {
+    let ticketId: string
+
+    beforeEach(async () => {
+      await deleteAuditDataWithPrefix(
+        AUDIT_BUCKET_NAME,
+        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}`
+      )
+      await deleteAuditDataWithPrefix(
+        ANALYSIS_BUCKET_NAME,
+        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}`
+      )
+      await copyAuditDataFromTestDataBucket(
+        AUDIT_BUCKET_NAME,
+        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}/01/${TEST_FILE_NAME}`,
+        TEST_FILE_NAME,
+        'GLACIER'
+      )
+      await copyAuditDataFromTestDataBucket(
+        AUDIT_BUCKET_NAME,
+        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}/02/${TEST_FILE_NAME}`,
+        TEST_FILE_NAME
+      )
+      ticketId = await createZendeskTicket(validMixRequestData)
+      await approveZendeskTicket(ticketId)
+    })
+
+    test('request for valid data in standard and glacier tier', async () => {
```

Comment:
As this is a valid request scenario, I'd rename  to `valid request with data in standard and glacier tier`